static/src/js/containers/DatepickerContainer/__tests__/DatepickerContainer.test.js

```diff
@@ -167,11 +274,15 @@ describe('DatepickerContainer component', () => {
     describe('when not given an custom time', () => {
       test('the "start" input passes the correct object', () => {
         const { enzymeWrapper, props } = setup()
-        const { onChange } = enzymeWrapper.instance()
+
+        enzymeWrapper.instance().picker.current = {}
+        enzymeWrapper.instance().picker.current.setState = jest.fn()
+
+        const { onInputBlur } = enzymeWrapper.instance()
         const { onSubmit } = props
         enzymeWrapper.setProps({ type: 'start' })
 
-        onChange(moment.utc('1988-09-03'))
```

Comment:
I dont see why this wouldve needed to be swapped to onInputBlur. Arent we still ultimately calling onChange with the values?